resources/brave-youtube-quality.js

```diff
@@ -0,0 +1,30 @@
+/// brave-youtube-quality.js
+(function() {

```

Comment:
@tapanmodh do you know if the same script works for desktop ? I think It should. We can create an issue for the desktop if we want to use the same feature on the desktop.